setup.py

```diff
@@ -22,6 +22,7 @@
         "microcosm>=2.12.0",
         "python-json-logger>=0.1.9",
         "requests[security]>=2.18.4",
+        "python-logstash-async>=2.3.0",
```

Comment:
@adamhadani Shouldn't need any async specific changes, at least not yet. We're using a synchronous handler provided by this same library because the async implementation conflicts with our process / threading that are used in different microservices.